### DISCUSSION TITLE
GitHub Education does not correctly identify my school in my school's mailing list

### DISCUSSION THREAD

**Discussion by Anghelgg** [community/community#64651] [GitHub Education] [answered]
### Hi everyone,

My email is not being accepted for the github student pack program, it is not being accepted, but it contains the domain of my university (buap.mx), since the university integrated the new email @alm.buap.mx, but it is not yet in the list.

**Comment by lab596** [comment]
Reach out to GitHub Support directly to explain the situation and provide them with the details of your university's new email domain. They should be able to assist you in getting your new email domain added to the list of eligible domains for the GitHub Student Pack program. You can contact GitHub Support through their [official support page](https://support.github.com/contact).

**Comment by KPlanisphere** [comment, marked as answer]
Contact GitHub Support and explain your university's change of email domain, providing the necessary details. They can help add your new domain to the GitHub Student Pack program's eligible domains list. Use their official support page to get in touch with them.
